accountClient/accountClient.go

```diff
@@ -46,6 +46,35 @@ func (c *E3dbAccountClient) InternalGetClientAccount(ctx context.Context, client
 	return result, err
 }
 
+func (c *E3dbAccountClient) RegisterClient(ctx context.Context, params ClientRegistrationRequest) (*ClientRegistrationResponse, error) {
+	var result *ClientRegistrationResponse
+	path := c.Host + "/" + AccountServiceBasePath + "/e3db/clients/register"
+	request, err := e3dbClients.CreateRequest("POST", path, params)
+	if err != nil {
+		return result, e3dbClients.NewError(err.Error(), path, 0)
+	}
+	resp, err := e3dbClients.ReturnE3dbServiceCall(c.E3dbAuthClient, ctx, request, &result)
+	if err != nil {
+		return result, e3dbClients.NewError(err.Error(), path, 0)
+	}
+	backupClient := resp.Header.Get("X-Backup-Client")
+	result.RootClientID = backupClient
+	return result, err
+}
+
+func (c *E3dbAccountClient) CreateRegistrationToken(ctx context.Context, accountServiceJWT string, params CreateRegTokenRequest) (*CreateRegTokenResponse, error) {
```

Comment:
either accountServiceJWT should be part of the struct for an `E3dbAccountClient` or it should be part of the params struct.

Reply:
specifically for testing, AFAIK the token only comes from a register request to account service?